Xamarin.Forms.Platform.iOS/Renderers/ListViewRenderer.cs

```diff
@@ -271,6 +271,9 @@ protected override void OnElementChanged(ElementChangedEventArgs<ListView> e)
 					_tableViewController = new FormsUITableViewController(e.NewElement, _usingLargeTitles);
 					SetNativeControl(_tableViewController.TableView);
 
+					if (Forms.IsiOS15OrNewer)
+						_tableViewController.TableView.SectionHeaderTopPadding = new nfloat(0);
```

Comment:
Although this fixes it, are we sure this is the actual issue? Is this a bug in iOS? Is our code causing this somehow only for iOS 15?

Reply:
We definitely need to analyze this well, we need more time. The PR is a Draft until we review everything better and we are really sure about everything.

Reply:
"error CS1061: 'UITableView' does not contain a definition for 'SectionHeaderTopPadding' and no accessible extension method 'SectionHeaderTopPadding' accepting a first argument of type 'UITableView' could be found (are you missing a using directive or an assembly reference?) [D:\agent\2\s\Xamarin.Forms.Platform.iOS\Xamarin.Forms.Platform.iOS.csproj]"

cc @rmarinho

Reply:
@jsuarezruiz Looks like the hosted build agents on Azure DevOps aren't yet configured with the latest Xamarin.iOS and/or Xcode versions. I can confirm the same result in a hosted pipeline.